src/yamlSettings.ts

```diff
@@ -54,6 +55,7 @@ export class SettingsState {
   schemaStoreSettings = [];
   customTags = [];
   schemaStoreEnabled = true;
+  schemaStoreUrl = 'https://www.schemastore.org/api/json/catalog.json';
```

Comment:
I think it worth to use `JSON_SCHEMASTORE_URL` constant to have single place where that URL defined.

Reply:
Fixed!